### DISCUSSION TITLE
After a proper restart can't find my data (data is not searchable).

### DISCUSSION THREAD

**Discussion by martinenkoEduard** [milvus-io/milvus#18381] [Q&A and General discussion]
I properly restarted milvus (releasing collection first, then issuing docker-compose down)
I loads collection just fine and say the right number of entries - but when I search for something it returns zero results.
![Screenshot 2022-07-23 092007](https://user-images.githubusercontent.com/108512856/180593179-4f331d4c-40ca-4d0a-a3c0-4df47a5a5fd9.png)
![Screenshot 2022-07-23 092206](https://user-images.githubusercontent.com/108512856/180593192-8643c270-e5af-4a3f-afdc-8b3d6b9b1689.png)
![Screenshot 2022-07-23 092216](https://user-images.githubusercontent.com/108512856/180593193-810da378-47b6-43b2-b320-38fc9df773d8.png)

**Comment by martinenkoEduard** [comment]
Found following error 
![image](https://user-images.githubusercontent.com/108512856/180593353-0934f599-0701-4f02-831f-6d6ee20d9d2e.png)

**Comment by martinenkoEduard** [comment]
milvus-standalone  | [2022/07/23 06:29:20.846 +00:00] [DEBUG] [time_recorder.go:78] ["search 434763921108238337(nq=1, k=10), msgID = 434780924561588281: historical search done, msgID = 434780924561588281 (0ms)"]
milvus-standalone  | [2022/07/23 06:29:20.846 +00:00] [DEBUG] [query_collection.go:1091] ["streaming search start"] [msgID=434780924561588281]
milvus-standalone  | [2022/07/23 06:29:20.846 +00:00] [DEBUG] [streaming.go:125] ["no partition specified, search all partitions"] [collectionID=434763921108238337] [vChannel=by-dev-rootcoord-dml_19_434763921108238337v1] ["all partitions"="[434763921108238338]"]
milvus-standalone  | [2022/07/23 06:29:20.846 +00:00] [DEBUG] [streaming.go:163] ["get segmentIDs by vChannel"] [collectionID=434763921108238337] [vChannel=by-dev-rootcoord-dml_19_434763921108238337v1] [partitionID=434763921108238338] [segmentIDs="[]"]
milvus-standalone  | [2022/07/23 06:29:20.846 +00:00] [DEBUG] [query_collection.go:1098] ["streaming search"] [msgID=434780924561588281] [collectionID=434763921108238337] ["searched dmChannel"=by-dev-rootcoord-dml_19_434763921108238337v1] ["searched partitionIDs"="[434763921108238338]"] ["searched segmentIDs"="[]"]
milvus-standalone  | [2022/07/23 06:29:20.846 +00:00] [DEBUG] [streaming.go:125] ["no partition specified, search all partitions"] [collectionID=434763921108238337] [vChannel=by-dev-rootcoord-dml_18_434763921108238337v0] ["all partitions"="[434763921108238338]"]
milvus-standalone  | [2022/07/23 06:29:20.846 +00:00] [DEBUG] [streaming.go:163] ["get segmentIDs by vChannel"] [collectionID=434763921108238337] [vChannel=by-dev-rootcoord-dml_18_434763921108238337v0] [partitionID=434763921108238338] [segmentIDs="[]"]
milvus-standalone  | [2022/07/23 06:29:20.846 +00:00] [DEBUG] [query_collection.go:1098] ["streaming search"] [msgID=434780924561588281] [collectionID=434763921108238337] ["searched dmChannel"=by-dev-rootcoord-dml_18_434763921108238337v0] ["searched partitionIDs"="[434763921108238338]"] ["searched segmentIDs"="[]"]
milvus-standalone  | [2022/07/23 06:29:20.846 +00:00] [DEBUG] [time_recorder.go:78] ["search 434763921108238337(nq=1, k=10), msgID = 434780924561588281: streaming search done, msgID = 434780924561588281 (0ms)"]
milvus-standalone  | [2022/07/23 06:29:20.846 +00:00] [DEBUG] [query_collection.go:1128] ["QueryNode Empty SearchResultMsg"] [collectionID=434763921108238337] [msgID=434780924561588281] [vChannels="[by-dev-rootcoord-dml_19_434763921108238337v1,by-dev-rootcoord-dml_18_434763921108238337v0]"] [sealedSegmentSearched="[]"]
milvus-standalone  | [2022/07/23 06:29:20.847 +00:00] [DEBUG] [task_scheduler.go:681] ["got a search result"] [role=proxy] [ReqID=434780924561588281]
milvus-standalone  | [2022/07/23 06:29:20.847 +00:00] [DEBUG] [task_scheduler.go:701] ["first receive search result of this task"] [role=proxy] [reqID=434780924561588281]
milvus-standalone  | [2022/07/23 06:29:20.847 +00:00] [DEBUG] [task_scheduler.go:717] ["process search result"] [role=proxy] [collection=faces] [reqID=434780924561588281] ["answer cnt"=1]
milvus-standalone  | [2022/07/23 06:29:20.847 +00:00] [DEBUG] [task_scheduler.go:609] ["check if result buf is ready to reduce"] [role=proxy] [MsgID=434780924561588281] [receivedVChansSet="[\"by-dev-rootcoord-dml_19_434763921108238337v1\",\"by-dev-rootcoord-dml_18_434763921108238337v0\"]"] [usedVChans="[\"by-dev-rootcoord-dml_19_434763921108238337v1\",\"by-dev-rootcoord-dml_18_434763921108238337v0\"]"] [receivedSealedSegmentIDsSet="[]"] [receivedGlobalSegmentIDsSet="[]"]
milvus-standalone  | [2022/07/23 06:29:20.847 +00:00] [DEBUG] [task_scheduler.go:720] ["process search result, ready to reduce"] [role=proxy] [reqID=434780924561588281]
milvus-standalone  | [2022/07/23 06:29:20.847 +00:00] [DEBUG] [task.go:1943] ["Proxy Search PostExecute stage1"] [len(filterSearchResults)=1]
milvus-standalone  | [2022/07/23 06:29:20.847 +00:00] [DEBUG] [time_recorder.go:78] ["searchTask PostExecute: Proxy Search PostExecute stage1 done (2ms)"]
milvus-standalone  | [2022/07/23 06:29:20.847 +00:00] [DEBUG] [time_recorder.go:78] ["searchTask PostExecute: decodeResultStart (0ms)"]
milvus-standalone  | [2022/07/23 06:29:20.847 +00:00] [DEBUG] [proxy_session_manager.go:108] ["success to send search result"] [node=24] [base="msg_type:SearchResult msgID:434780924561588281 timestamp:434780965652922374 sourceID:24 "]
milvus-standalone  | [2022/07/23 06:29:20.847 +00:00] [DEBUG] [task.go:1735] [decodeSearchResults] [lenOfSearchResults=1]
milvus-standalone  | [2022/07/23 06:29:20.847 +00:00] [DEBUG] [time_recorder.go:78] ["search 434763921108238337(nq=1, k=10), msgID = 434780924561588281: publish empty search result done, msgID = 434780924561588281 (1ms)"]
milvus-standalone  | [2022/07/23 06:29:20.847 +00:00] [DEBUG] [time_recorder.go:78] ["search 434763921108238337(nq=1, k=10), msgID = 434780924561588281: all done, msgID = 434780924561588281 (1ms)"]
milvus-standalone  | [2022/07/23 06:29:20.847 +00:00] [DEBUG] [time_recorder.go:78] ["receiveQueryMsg 434780924561588281: operation done (2ms)"]
milvus-standalone  | [2022/07/23 06:29:20.847 +00:00] [DEBUG] [query_collection.go:589] ["do query done in receiveQueryMsg"] [collectionID=434763921108238337] [msgID=434780924561588281] [msgType=search]
milvus-standalone  | [2022/07/23 06:29:20.847 +00:00] [DEBUG] [time_recorder.go:78] ["receiveQueryMsg 434780924561588281: all done (2ms)"]
milvus-standalone  | [2022/07/23 06:29:20.847 +00:00] [DEBUG] [task.go:1740] [decodeSearchResults] [i=0] [len(SlicedBob)=0]
milvus-standalone  | [2022/07/23 06:29:20.847 +00:00] [DEBUG] [task.go:1754] [decodeSearchResults] [lenOfResults=0]
milvus-standalone  | [2022/07/23 06:29:20.847 +00:00] [DEBUG] [time_recorder.go:78] ["decodeSearchResults: done (0ms)"]
milvus-standalone  | [2022/07/23 06:29:20.847 +00:00] [DEBUG] [task.go:1963] ["Proxy Search PostExecute stage2"] [len(validSearchResults)=0]
milvus-standalone  | [2022/07/23 06:29:20.848 +00:00] [DEBUG] [task.go:1966] ["Proxy Search PostExecute stage2 failed"] [filterReason="empty search result\n"]
milvus-standalone  | [2022/07/23 06:29:20.848 +00:00] [DEBUG] [time_recorder.go:78] ["searchTask PostExecute: done (2ms)"]
milvus-standalone  | [2022/07/23 06:29:20.848 +00:00] [DEBUG] [impl.go:2500] ["Search done"] [traceID=16426306db8aaf2c] [role=proxy] [msgID=434780924561588281] [db=] [collection=faces] [partitions="[]"] [dsl=] [len(PlaceholderGroup)=2060] [OutputFields="[]"] [search_params="[{\"key\":\"anns_field\",\"value\":\"embeddings\"},{\"key\":\"topk\",\"value\":\"10\"},{\"key\":\"metric_type\",\"value\":\"L2\"},{\"key\":\"params\",\"value\":\"{}\"},{\"key\":\"round_decimal\",\"value\":\"-1\"}]"] [travel_timestamp=0] [guarantee_timestamp=434780964347183104]
milvus-standalone  | [2022/07/23 06:29:25.622 +00:00] [DEBUG] [query_coord.go:530] ["loadBalanceSegmentLoop: memory usage rate of all online QueryNode"] ["mem rate"="{\"19\":0.45333381727364813}"]
milvus-standalone  | [2022/07/23 06:29:25.622 +00:00] [WARN] [query_coord.go:532] ["loadBalanceSegmentLoop: there are too few available query nodes to balance"] [onlineNodeIDs="[19]"] [availableNodeIDs="[19]"]

**Reply by martinenkoEduard** [reply]
milvus-standalone  | [2022/07/23 06:30:39.022 +00:00] [DEBUG] [query_collection.go:631] ["get query message from unsolvedMsg"] [collectionID=434763921108238337] [msgID=434780924561588283] [reqTime_p=2022/07/23 06:30:38.821 +00:00] [serviceTime_p=2022/07/23 06:30:38.672 +00:00] [guaranteeTime_l=434780986099892226] [serviceTime_l=434780986060832769]
milvus-standalone  | [2022/07/23 06:30:39.022 +00:00] [DEBUG] [query_collection.go:350] ["check if query timeout"] [collectionID=434763921108238337] [msgID=434780924561588283] [TimeoutTs=0] [curTime=434780986152583168] [timeoutTsPhysical=1970/01/01 00:00:00.000 +00:00] [curTimePhysical=2022/07/23 06:30:39.022 +00:00]
milvus-standalone  | [2022/07/23 06:30:39.023 +00:00] [DEBUG] [query_collection.go:656] ["query node::doUnsolvedMsg: add to unsolvedMsg"] [collectionID=434763921108238337] [sm.BeginTs=2022/07/23 06:30:38.821 +00:00] [serviceTime=2022/07/23 06:30:38.672 +00:00] ["delta seconds"=0] [msgID=434780924561588283]
milvus-standalone  | [2022/07/23 06:30:39.023 +00:00] [DEBUG] [query_collection.go:631] ["get query message from unsolvedMsg"] [collectionID=434763921108238337] [msgID=434780924561588283] [reqTime_p=2022/07/23 06:30:38.821 +00:00] [serviceTime_p=2022/07/23 06:30:38.871 +00:00] [guaranteeTime_l=434780986099892226] [serviceTime_l=434780986112999425]
milvus-standalone  | [2022/07/23 06:30:39.023 +00:00] [DEBUG] [query_collection.go:350] ["check if query timeout"] [collectionID=434763921108238337] [msgID=434780924561588283] [TimeoutTs=0] [curTime=434780986152845312] [timeoutTsPhysical=1970/01/01 00:00:00.000 +00:00] [curTimePhysical=2022/07/23 06:30:39.023 +00:00]
milvus-standalone  | [2022/07/23 06:30:39.023 +00:00] [DEBUG] [query_collection.go:674] ["doing search in doUnsolvedMsg..."] [collectionID=434763921108238337] [msgID=434780924561588283]
milvus-standalone  | [2022/07/23 06:30:39.023 +00:00] [DEBUG] [query_collection.go:1314] ["historical retrieve start"] [msgID=434780924561588283]
milvus-standalone  | [2022/07/23 06:30:39.023 +00:00] [DEBUG] [query_collection.go:1320] ["historical retrieve"] [msgID=434780924561588283] [collectionID=434763921108238337] ["retrieve partitionIDs"="[434763921108238338]"] ["retrieve segmentIDs"="[]"]
milvus-standalone  | [2022/07/23 06:30:39.023 +00:00] [DEBUG] [time_recorder.go:78] ["retrieve 434763921108238337: historical retrieve done, msgID = 434780924561588283 (0ms)"]
milvus-standalone  | [2022/07/23 06:30:39.023 +00:00] [DEBUG] [query_collection.go:1324] ["streaming retrieve start"] [msgID=434780924561588283]
milvus-standalone  | [2022/07/23 06:30:39.023 +00:00] [DEBUG] [query_collection.go:1330] ["streaming retrieve"] [msgID=434780924561588283] [collectionID=434763921108238337] ["retrieve partitionIDs"="[434763921108238338]"] ["retrieve segmentIDs"="[]"]
milvus-standalone  | [2022/07/23 06:30:39.023 +00:00] [DEBUG] [time_recorder.go:78] ["retrieve 434763921108238337: streaming retrieve done, msgID = 434780924561588283 (0ms)"]
milvus-standalone  | [2022/07/23 06:30:39.023 +00:00] [DEBUG] [query_collection.go:1417] ["skip duplicated query result"] [count=0]
milvus-standalone  | [2022/07/23 06:30:39.023 +00:00] [DEBUG] [time_recorder.go:78] ["retrieve 434763921108238337: merge result done, msgID = 434780924561588283 (0ms)"]
milvus-standalone  | [2022/07/23 06:30:39.024 +00:00] [DEBUG] [task_scheduler.go:742] ["got a retrieve result"] [role=proxy] [ReqID=434780924561588283]
milvus-standalone  | [2022/07/23 06:30:39.024 +00:00] [DEBUG] [task_scheduler.go:762] ["first receive retrieve result of this task"] [role=proxy] [reqID=434780924561588283]
milvus-standalone  | [2022/07/23 06:30:39.024 +00:00] [DEBUG] [task_scheduler.go:778] ["process retrieve result"] [role=proxy] [collection=faces] [reqID=434780924561588283] ["answer cnt"=1]
milvus-standalone  | [2022/07/23 06:30:39.024 +00:00] [DEBUG] [task_scheduler.go:609] ["check if result buf is ready to reduce"] [role=proxy] [MsgID=434780924561588283] [receivedVChansSet="[\"by-dev-rootcoord-dml_19_434763921108238337v1\",\"by-dev-rootcoord-dml_18_434763921108238337v0\"]"] [usedVChans="[\"by-dev-rootcoord-dml_19_434763921108238337v1\",\"by-dev-rootcoord-dml_18_434763921108238337v0\"]"] [receivedSealedSegmentIDsSet="[]"] [receivedGlobalSegmentIDsSet="[]"]
milvus-standalone  | [2022/07/23 06:30:39.024 +00:00] [DEBUG] [task_scheduler.go:781] ["process retrieve result, ready to reduce"] [role=proxy] [reqID=434780924561588283]
milvus-standalone  | [2022/07/23 06:30:39.024 +00:00] [DEBUG] [task.go:2386] ["skip duplicated query result"] [count=0]
milvus-standalone  | [2022/07/23 06:30:39.024 +00:00] [INFO] [task.go:2442] ["Query result is nil"] [requestID=434780924561588283] [requestType=query]
milvus-standalone  | [2022/07/23 06:30:39.024 +00:00] [DEBUG] [time_recorder.go:78] ["queryTask PostExecute: done (167ms)"]
milvus-standalone  | [2022/07/23 06:30:39.024 +00:00] [DEBUG] [impl.go:2718] ["Query done"] [traceID=72cef0c9566e2be8] [role=proxy] [MsgID=434780924561588283] [BeginTs=434780986099892226] [EndTs=434780986099892226] [db=] [collection=faces] [partitions="[]"]
milvus-standalone  | [2022/07/23 06:30:39.025 +00:00] [DEBUG] [proxy_session_manager.go:126] ["success to send retrieve result"] [node=24] [base="msg_type:RetrieveResult msgID:434780924561588283 sourceID:24 "]
milvus-standalone  | [2022/07/23 06:30:39.025 +00:00] [DEBUG] [query_collection.go:1366] ["QueryNode publish RetrieveResultMsg"] [msgID=434780924561588283] [vChannels="[by-dev-rootcoord-dml_19_434763921108238337v1,by-dev-rootcoord-dml_18_434763921108238337v0]"] [collectionID=434763921108238337] [sealedSegmentRetrieved="[]"]
milvus-standalone  | [2022/07/23 06:30:39.025 +00:00] [DEBUG] [time_recorder.go:78] ["retrieve 434763921108238337: all done, msgID = 434780924561588283 (2ms)"]
milvus-standalone  | [2022/07/23 06:30:39.025 +00:00] [DEBUG] [query_collection.go:706] ["do query done in doUnsolvedMsg"] [collectionID=434763921108238337] [msgID=434780924561588283]
milvus-standalone  | [2022/07/23 06:30:39.025 +00:00] [DEBUG] [query_collection.go:711] ["doUnsolvedMsg: do query done"] ["num of query msg"=1]

**Comment by martinenkoEduard** [comment]
When I try to create index - i get 
milvus-standalone  | [2022/07/23 06:33:20.324 +00:00] [INFO] [services.go:928] ["received get flush state request"] [segmentIDs="[434763927989256193,434763931240366081]"] [len=2]
milvus-standalone  | [2022/07/23 06:33:20.324 +00:00] [INFO] [services.go:951] ["[flush state] unflushed segment ids"] [segmentIDs="[434763927989256193,434763931240366081]"] [len=2]
milvus-standalone  | [2022/07/23 06:33:20.324 +00:00] [INFO] [impl.go:3906] ["received get flush state response"] [response="status:<> "]
milvus-standalone  | [2022/07/23 06:33:20.827 +00:00] [INFO] [impl.go:3892] ["received get flush state request"] [request="segmentIDs:434763927989256193 segmentIDs:434763931240366081 "]
milvus-standalone  | [2022/07/23 06:33:20.828 +00:00] [INFO] [services.go:928] ["received get flush state request"] [segmentIDs="[434763927989256193,434763931240366081]"] [len=2]
milvus-standalone  | [2022/07/23 06:33:20.828 +00:00] [INFO] [services.go:951] ["[flush state] unflushed segment ids"] [segmentIDs="[434763927989256193,434763931240366081]"] [len=2]
milvus-standalone  | [2022/07/23 06:33:20.828 +00:00] [INFO] [impl.go:3906] ["received get flush state response"] [response="status:<> "]
milvus-standalone  | [2022/07/23 06:33:21.333 +00:00] [INFO] [impl.go:3892] ["received get flush state request"] [request="segmentIDs:434763927989256193 segmentIDs:434763931240366081 "]
milvus-standalone  | [2022/07/23 06:33:21.334 +00:00] [INFO] [services.go:928] ["received get flush state request"] [segmentIDs="[434763927989256193,434763931240366081]"] [len=2]
milvus-standalone  | [2022/07/23 06:33:21.334 +00:00] [INFO] [services.go:951] ["[flush state] unflushed segment ids"] [segmentIDs="[434763927989256193,434763931240366081]"] [len=2]
milvus-standalone  | [2022/07/23 06:33:21.334 +00:00] [INFO] [impl.go:3906] ["received get flush state response"] [response="status:<> "]
milvus-standalone  | [2022/07/23 06:33:21.837 +00:00] [INFO] [impl.go:3892] ["received get flush state request"] [request="segmentIDs:434763927989256193 segmentIDs:434763931240366081 "]
milvus-standalone  | [2022/07/23 06:33:21.838 +00:00] [INFO] [services.go:928] ["received get flush state request"] [segmentIDs="[434763927989256193,434763931240366081]"] [len=2]
milvus-standalone  | [2022/07/23 06:33:21.838 +00:00] [INFO] [services.go:951] ["[flush state] unflushed segment ids"] [segmentIDs="[434763927989256193,434763931240366081]"] [len=2]
milvus-standalone  | [2022/07/23 06:33:21.838 +00:00] [INFO] [impl.go:3906] ["received get flush state response"] [response="status:<> "]
milvus-standalone  | [2022/07/23 06:33:22.342 +00:00] [INFO] [impl.go:3892] ["received get flush state request"] [request="segmentIDs:434763927989256193 segmentIDs:434763931240366081 "]
milvus-standalone  | [2022/07/23 06:33:22.343 +00:00] [INFO] [services.go:928] ["received get flush state request"] [segmentIDs="[434763927989256193,434763931240366081]"] [len=2]
milvus-standalone  | [2022/07/23 06:33:22.343 +00:00] [INFO] [services.go:951] ["[flush state] unflushed segment ids"] [segmentIDs="[434763927989256193,434763931240366081]"] [len=2]
milvus-standalone  | [2022/07/23 06:33:22.344 +00:00] [INFO] [impl.go:3906] ["received get flush state response"] [response="status:<> "]
milvus-standalone  | [2022/07/23 06:33:22.846 +00:00] [INFO] [impl.go:3892] ["received get flush state request"] [request="segmentIDs:434763927989256193 segmentIDs:434763931240366081 "]
milvus-standalone  | [2022/07/23 06:33:22.847 +00:00] [INFO] [services.go:928] ["received get flush state request"] [segmentIDs="[434763927989256193,434763931240366081]"] [len=2]
milvus-standalone  | [2022/07/23 06:33:22.847 +00:00] [INFO] [services.go:951] ["[flush state] unflushed segment ids"] [segmentIDs="[434763927989256193,434763931240366081]"] [len=2]
milvus-standalone  | [2022/07/23 06:33:22.847 +00:00] [INFO] [impl.go:3906] ["received get flush state response"] [response="status:<> "]
milvus-standalone  | [2022/07/23 06:33:23.351 +00:00] [INFO] [impl.go:3892] ["received get flush state request"] [request="segmentIDs:434763927989256193 segmentIDs:434763931240366081 "]
milvus-standalone  | [2022/07/23 06:33:23.352 +00:00] [INFO] [services.go:928] ["received get flush state request"] [segmentIDs="[434763927989256193,434763931240366081]"] [len=2]
milvus-standalone  | [2022/07/23 06:33:23.352 +00:00] [INFO] [services.go:951] ["[flush state] unflushed segment ids"] [segmentIDs="[434763927989256193,434763931240366081]"] [len=2]
milvus-standalone  | [2022/07/23 06:33:23.353 +00:00] [INFO] [impl.go:3906] ["received get flush state response"] [response="status:<> "]
milvus-standalone  | [2022/07/23 06:33:23.857 +00:00] [INFO] [impl.go:3892] ["received get flush state request"] [request="segmentIDs:434763927989256193 segmentIDs:434763931240366081 "]
milvus-standalone  | [2022/07/23 06:33:23.858 +00:00] [INFO] [services.go:928] ["received get flush state request"] [segmentIDs="[434763927989256193,434763931240366081]"] [len=2]
milvus-standalone  | [2022/07/23 06:33:23.858 +00:00] [INFO] [services.go:951] ["[flush state] unflushed segment ids"] [segmentIDs="[434763927989256193,434763931240366081]"] [len=2]
milvus-standalone  | [2022/07/23 06:33:23.859 +00:00] [INFO] [impl.go:3906] ["received get flush state response"] [response="status:<> "]

**Comment by martinenkoEduard** [comment]
Also before I encountered a similar problem - but I got only partial lost.
For instance ids from 1 to 4000 were lost. but some were saved.

**Comment by martinenkoEduard** [comment]
Could it be because of consistency level?

**Comment by ziqi521** [comment]
也是上面的问题，咨询下，how to slove？thks

**Reply by martinenkoEduard** [reply]
I accidentally found a temporary solution/a crutch.
Do only once
print(f"count: {collection.num_entities}")

And lost entries will appear again.

**Reply by ziqi521** [reply]
thank you！very good。

**Comment by yhmo** [comment]
When you call insert(), the data will be process in this sequence:
```
    call insert()                                     flush()
         |                                              |
client ----> proxy node ----> pulsar --|--> data node ----> minio
                                       |---> query node

```

After the insert() is called, the data firstly pass to proxy node, then proxy node send the data to pulsar service. Then query node and data node subscribe the data from pulsar. Before the data is consumed by query node, the data is invisible to user(not searchable).
If a search action requires to see the data, then the action need to set a consistency level to make sure the data is consumed by query node. Set consistency level=Strong can make sure the data of insert actions before the search action is consumed by query node.
